jobs/rootfs-builder.jpl

```diff
@@ -123,10 +123,34 @@ node("debos && docker") {
             }
         }
 
+        stage("scp") {
```

Comment:
Maybe `stage("Upload")` would be a bit better, to keep the same name as before. That way, all the roofs-builder jobs will appear with the same pipeline steps in Jenkins.